src/app/styles/normalize.scss

```diff
@@ -8,10 +8,12 @@ body,
 
 html {
     scroll-behavior: smooth;
+    scrollbar-width: none;
 }
 
 body {
     margin: 0;
+    overflow-x: hidden;
```

Comment:
Please, do not change normalize file to change some of our styles. Idea of normalizition of syles is to reset the default styles of HTML elements across different web browsers. So, if u want to fix some normalizition - it is okey, but i'm not shure we need to hide for body content with overflow-x: hidden; Try to do this on element u need.